src/no-wallaby-comment.spec.ts

```diff
@@ -0,0 +1,97 @@
+// no-wallaby-comment.spec.ts
+
+/*
+ * Copyright (c) 2022-2023 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+import { RuleTester } from 'eslint';
+
+import rule from './no-wallaby-comment';
+
+const LINE_WITH_NO_COMMENTS = `const NOT_A_SECRET = "A template that isn't a secret.";`;
+
+const LINE_ONE_WITH_MULTIPLE_COMMENTS = `
+// file.skip const NOT_A_SECRET = "A template that isn't a secret.";
```

Comment:
file.skip and file.only are normally on lines by themselves, not in front of commented code.  think we need to test that scenario instead (tbh not sure what Wallabys behavior is here)